code/game/turfs/space/space.dm

```diff
@@ -85,30 +85,18 @@
 			user << "\red The plating is going to need some support."
 	return
 
-
-// Ported from unstable r355
-
 /turf/space/Entered(atom/movable/A as mob|obj)
 	..()
-	if ((!(A) || src != A.loc))	return
-
-	if(transition)
-
-		if(A.z > MAX_Z)	return //for away missions
-
-		if(destination_x)
-			A.x = destination_x
-
-		if(destination_y)
-			A.y = destination_y
-
-		var/obj/item/flag/nation/fukkenflag = A.GetTypeInAllContents(/obj/item/flag/nation)
-		if(fukkenflag)
-			fukkenflag.loc = fukkenflag.startloc

```

Comment:
I intend to fully purge nations from the game soon™ because it has its tentacles in all parts of the code. Plus - we don't run nations anymore nor should we, so I don't see the point in continuing to support it.